docker-compose.enterprise.yml

```diff
@@ -103,6 +103,9 @@ services:
         environment:
             DEVICECONNECT_ENABLE_AUDIT: 1
 
+    mender-generate-delta-worker:
+        image: registry.mender.io/mendersoftware/generate-delta-worker:mender-master
+
```

Comment:
This is wrong, because mender-generate-delta-worker misses all the other configuration settings (see mender-devicemonitor as an example). Please, fix it and try to run integration locally.